src/main/kotlin/org/jetbrains/kotlinx/jupyter/repl/CellExecutor.kt

```diff
@@ -17,6 +17,7 @@ interface CellExecutor : ExecutionHost {
         processAnnotations: Boolean = true,
         processMagics: Boolean = true,
         invokeAfterCallbacks: Boolean = true,
+        currentCellId: Int = 1,
```

Comment:
First cell has `ID = 0`, isn't it? If so, `0` is a better default value. Or even `-1` to better distinguish cases of set and not set value

Reply:
-1 seems potentially more useful, yeah